app/subscriber/src/features/filter-media/FilterMediaLanding.tsx

```diff
@@ -0,0 +1,146 @@
+import { useFilterOptions } from 'components/navbar';
+import { PageSection } from 'components/section';
+import React from 'react';
+import { useContent } from 'store/hooks';
+import { Col, ISourceModel, Row, Show } from 'tno-core';
+
+import { MediaFilterTypes } from './constants';
+import { FilterMedia } from './FilterMedia';
+import * as styled from './styled';
+import { alphabetArray } from './utils';
```

Comment:
should we be using a constant for this?  This can lead to a situation where letters show up for which there are no options associated.